util/fusermount.c

```diff
@@ -1468,6 +1468,7 @@ int main(int argc, char *argv[])
 
 	if (!auto_unmount) {
 		free(mnt);
+		free((void*) type);
```

Comment:
This is only here to make address sanitizer happy.  I've moved it into its own pull request at https://github.com/libfuse/libfuse/pull/785 so we can remove it here once it's landed in master.